includes/utilities.inc

```diff
@@ -237,7 +246,7 @@ function islandora_entities_get_related_pids($identifier, $title, $type) {
       $results[$choice['PID']] = array(
         'PID' => $choice['PID'],
         'title' => $choice['solr_doc']['dc.title'],
-        'authors' => $choice['solr_doc']['mods_name_personal_displayForm_mt'],
+        'authors' => isset($choice['solr_doc'][$author_field]) ? $choice['solr_doc'][$author_field] : array(),
```

Comment:
Regarding ISLANDORA-2063. This PR does not solve the errors that are appearing on the citation tab. Those errors were introduced when we added the IslandoraSolrQueryProcessor to this module (ISLANDORA-1594) and that ticket has been reopened, and another PR should be on its way shortly! They are because the field that you're looking for (formerly mods_name_personal_displayForm_mt, and/or dc.title) are not in the list of fields you configured solr searches to display, and you configured Solr to "only return those fields" (this is a huge design flaw IMO). 

Also, annoyingly, dc.title (line 248) is never used. This function is only called on line 47 of citation_tab.inc, and there they use the object's label instead of the title that we found here as the text of the link. So the `'title'` line could technically vanish altogether.